kube/minkube/init/init-pod.yaml

```diff
@@ -16,9 +15,8 @@ spec:
     - name: data-volume
       mountPath: /data
     # resource limits needs to be discussed
-    resources: 
+    resources:
```

Comment:
I think it is not related to copyright fixes and can be separate commit?

Reply:
No that's just whitespace automatically removed.  It's normal for trailing whitespace to cause git noise.